### ISSUE TITLE
Filter by tag

### ISSUE THREAD

**Issue by luksan47** [luksan47/mars#432]
For example clicking on a collegist role on the user management page will show only the collegist.
We could use chips for this:
https://materializecss.com/chips.html

**Comment by kdmnk** [comment]
> We could use chips for this:

I would stay with badges instead of introducing an other element for the same thing.

**Comment by luksan47** [comment]
We might want to consider moving to chips from badges, I think badges were not designed for what we are using them.

**Comment by kdmnk** [comment]
Maybe, but badges are smaller and looks better. But probably we can modify chips to be colorful and smaller also.
I still think that we can stay with the badges, but if the chips provide more functionality, we should change.
________________________________

**Comment by luksan47** [comment]
Yeah, we can keep it, we already have everything we need with tags as well.
